pom.xml

```diff
@@ -40,6 +40,7 @@
     <xalan.version>2.7.1</xalan.version>
     <xercesImpl.version>2.9.1</xercesImpl.version>
     <pentaho-cgg-plugin.version>${project.version}</pentaho-cgg-plugin.version>
+    <mail.version>1.4.4</mail.version>
```

Comment:
In analyzer it is using `<mail.version>1.4.1</mail.version>` (https://github.com/pentaho/pentaho-analyzer/blob/482e0ae5ea420e06f4ef3c4e50cfff75d185ede6/pom.xml#L96). In OSGi it is using `1.4.4`, but that shouldn't be available in the non-OSGi side.